docs/content/dev/documentation.md

```diff
@@ -1,18 +1,15 @@
 # Documentation
 
-Our documentation is build with [mkdocs](https://www.mkdocs.org).
+Our documentation is build with [mkdocs][mkdocs].
 
 ## Writing
 
 All documentation files are found in the `docs/content` directory of
 the [hedgedoc/hedgedoc repo](https://github.com/hedgedoc/hedgedoc). These files are just normal markdown files with
 nothing special about them.
 
-The configuration for mkdocs lies in the `docs` folder in a file called `mkdocs.yml`. With that file the theme and menu
-
-- amoung others - can be configured.
-  **Please note:** Any new files need to be linked to by other files or put in the navigation or the files will be very
-  hard to find on the documentation website.
+The configuration for mkdocs lies in the `docs` folder in a file called `mkdocs.yml`. With that file the theme and menu - among others - can be configured.
+  **Please note:** Any new files need to be linked to by other files or put in the navigation, or the files will be very hard to find on the documentation website.
```

Comment:
```suggestion
  **Please note:** Any new files need to be linked to by other files or put in the navigation, otherwise the files will be very hard to find on the documentation website.
```

Reply:
fixed